tests/auxtel/test_latiss.py

```diff
@@ -71,7 +71,9 @@ async def test_take_flats(self):
             nflats = 4
             exptime = 1.0
             filter_id = 1
+            filter_name = f'filter{filter_id}'
```

Comment:
Do you mind using double-quotes? `f"filter{filter_id}"`, it is what is used throughout the code. 😬